test/beacon/Voucher.test.ts

```diff
@@ -405,5 +409,151 @@ describe('Voucher', function () {
                 ),
             ).to.be.revertedWith('IexecPocoMock: Failed to sponsorMatchOrders');
         });
+        describe('Match orders boost', async function () {
```

Comment:
```suggestion

        describe('Match orders boost', async function () {
```

Reply:
Not sure :thinking: 
My comment is to add a blank line before describe line.

Reply:
Oups was for : https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/16#discussion_r1584778727